### ISSUE TITLE
System.TypeInitializationException:  System.IO.FileLoadException: The path is too long after being fully qualified.

### ISSUE THREAD

**Issue by brcaswell** [SiliconStudio/xenko#339]
1.3.3 Launcher
1.3.4-beta project

In addition to the build error, there is a build warning.

```
2>------ Build started: Project: ArenaGame.Windows, Configuration: Debug Any CPU ------
2>  Consider app.config remapping of assembly "System.Runtime, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "4.0.0.0" [C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.5\Facades\System.Runtime.dll] to Version "4.0.10.0" [] to solve conflict and get rid of warning.
2>C:\Program Files (x86)\MSBuild\12.0\bin\Microsoft.Common.CurrentVersion.targets(1697,5): warning MSB3276: Found conflicts between different versions of the same dependent assembly. Please set the "AutoGenerateBindingRedirects" property to true in the project file. For more information, see http://go.microsoft.com/fwlink/?LinkId=294190.
2>  Patch for assembly [ArenaGame, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]
2>  ArenaGame.Windows -> C:\Users\Brett\Documents\Paradox Projects\ArenaGame\ArenaGame\Bin\Windows\Debug\ArenaGame.exe
2>  Unexpected exception: System.TypeInitializationException: The type initializer for '<Module>' threw an exception. ---> System.IO.FileLoadException: The path is too long after being fully qualified.  Make sure the full path is less than 260 characters and the directory name is less than 248 characters.
2>     at SiliconStudio.DataSerializers.SiliconStudioAssetsCompilerAppSerializerFactory.Initialize()
2>     at .cctor()
2>     --- End of inner exception stack trace ---
2>     at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor)
2>     at System.Reflection.RuntimeMethodInfo.UnsafeInvokeInternal(Object obj, Object[] parameters, Object[] arguments)
2>     at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
2>     at SiliconStudio.ExecServer.AppDomainShadow.AssemblyLoaderCallback.Run()
2>     at System.AppDomain.DoCallBack(CrossAppDomainDelegate callBackDelegate)
2>     at SiliconStudio.ExecServer.AppDomainShadow.Run(String workingDirectory, String[] args, IServerLogger logger)
2>C:\Users\Brett\AppData\Local\Programs\Silicon Studio\Paradox\GamePackages\Paradox.1.3.4-beta\Targets\Paradox.targets(219,5): error MSB3073: The command ""C:\Users\Brett\AppData\Local\Programs\Silicon Studio\Paradox\GamePackages\Paradox.1.3.4-beta\Bin\Windows-Direct3D11\SiliconStudio.Assets.CompilerClient.exe"  --disable-auto-compile --project-configuration "Debug" --platform=Windows --graphics-platform=Direct3D11 --profile=Windows --project-configuration=Debug --output-path="C:\Users\Brett\Documents\Paradox Projects\ArenaGame\ArenaGame\Bin\Windows\Debug\data" --build-path="C:\Users\Brett\Documents\Paradox Projects\ArenaGame\ArenaGame\ArenaGame.Windows\..\obj\data" --package-file="C:\Users\Brett\Documents\Paradox Projects\ArenaGame\ArenaGame\ArenaGame.Windows\..\ArenaGame.pdxpkg" --log-pipe=""" exited with code 1.
========== Build: 1 succeeded, 1 failed, 0 up-to-date, 0 skipped ==========
```

utilizing 3rd party target builds is a fairly new experience for me.

I see this issue has been raised [before][1]; I'm not sure why you closed it without actually addressing the underline issue - I created the project via your launcher (which I downloaded last night):

>Not sure why yours is still going to an old installation path

is a comment that seems to improperly scope the issue.

Note: when I installed the launcher, I designated the install to be for 'me only' (is that part of the consideration?).

> You probably have to manually fix the SiliconStudioParadoxDir environment variable.

Do you set the environment variable via regedit or command-line console (is it a build shell?)

   [1]: https://github.com/SiliconStudio/paradox/issues/328

**Comment by boriscallens** [comment]
My guess is the 'me only' option dictates that it should be installed in your user folder. This makes it usually ends up in a rather long path.

Installing the studio in a shorter path (like e.g. C:\Program Files (x86)) could fix that I think.

**Comment by Benlitz** [comment]
as @boriscallens said, installing in a shorter path, such as the "All users" path (in Program Files - not x86 by the way, Xenko is a x64 application) should workaround this issue.
We are currently investigating if we can properly fix it (the process is probably building a too long relative path from your installation folder that we might be able to simplify).

**Comment by xen2** [comment]
Fixed in 1.6.7!
